ember-cli-build.js

```diff
@@ -20,7 +20,7 @@ module.exports = function(defaults) {
   // please specify an object with the list of modules as keys
   // along with the exports of each module as its value.
 
-  app.import('bower_components/normalize-css/normalize.css');
+  app.import('node_modules/normalize.css/normalize.css');
```

Comment:
this doesn't work yet. `this.import()` for files in `node_modules` will only work starting with Ember CLI v2.15